pages/portfolio.html

```diff
@@ -4,9 +4,34 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Cabebe's Portfolio</title>
+  <link href="/styles/styles.css" rel="stylesheet">
```

Comment:
If you moved the index.html file to the root of the projects the paths of the files would like the following:
```suggestion
  <link href="./styles/styles.css" rel="stylesheet">
```

Reply:
The file paths would need to change for the rest of your files too.